scripts/linux/runE2ETest.sh

```diff
@@ -979,7 +987,9 @@ LOADGEN2_TRANSPORT_TYPE="${LOADGEN2_TRANSPORT_TYPE:-amqp}"
 LOADGEN3_TRANSPORT_TYPE="${LOADGEN3_TRANSPORT_TYPE:-mqtt}"
 LOADGEN4_TRANSPORT_TYPE="${LOADGEN4_TRANSPORT_TYPE:-mqtt}"
 if [[ "${TEST_NAME,,}" == "longhaul" ]]; then
+    DESIRED_MODULES_TO_RESTART_CSV="${DESIRED_MODULES_TO_RESTART_CSV:-,}"
```

Comment:
did you try empty string as default value?

Reply:
yes